stock_unique_serial/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright NuoBiT Solutions, S.L. (<https://www.nuobit.com>)
+# Eric Antones <eantones@nuobit.com>
+# Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Stock Unique Serial",
+    "summary": "This module ensures that the serial number have only one "
+    "stock location with quantity",
+    "version": "14.0.1.0.0",
+    "author": "NuoBiT Solutions, S.L., Eric Antones",
```

Comment:
```suggestion
    "author": "NuoBiT Solutions, S.L.",
```
Remember this, it's always the same